syz-manager/manager.go

```diff
@@ -653,6 +691,11 @@ func (mgr *Manager) saveRepro(res *repro.Result) {
 		}
 	}
 
+	// Apend this repro to repro list to send to hub if it didn't come from hub originally.
+	if !hub {
+		mgr.repros = append(mgr.repros, prog)
```

Comment:
This looks like a race. mgr.repros is accessed from vmLoop and from hubSync without synchronization. Try -race flag, it should point to all such races.